client/src/connection/itc/script.ts

```diff
@@ -12,6 +12,42 @@ export const scriptContent = `
 class SASRunner{
   [System.__ComObject] $objSAS
 
+  SASRunner([string]$interopPath) {
+    try {
+      $interopDir = $this.GetInteropDirectory($interopPath)
+      Add-Type -Path "$interopDir\\SASInterop.dll"
+      Add-Type -Path "$interopDir\\SASOManInterop.dll"
+    } catch {
+      Write-Error "${ERROR_START_TAG}Init error${ERROR_END_TAG}"
```

Comment:
I see that other catch blocks print error details, do you want to do the same?
https://github.com/sassoftware/vscode-sas-extension/blob/main/client/src/connection/itc/script.ts#L106
```suggestion
      Write-Error "${ERROR_START_TAG}Init error: $_${ERROR_END_TAG}"
```

Reply:
Addressed in https://github.com/sassoftware/vscode-sas-extension/pull/1409/commits/a2b4f0e8bd572d3067f6a9b2c1023d45aa88fde4